lib/screen/interactive_postcard/postcard_journey.dart

```diff
@@ -0,0 +1,311 @@
+    final assetToken = widget.assetToken;
+    Navigator.of(context).pushNamed(AppRouter.tripDetailPage,
+        arguments: TripDetailPayload(
+          stampIndex: index,
+          travelsInfo: listTravelInfo,
+          assetToken: assetToken,
+        ));
+  }
+
+  Widget _travelWidget(TravelInfo travelInfo,
+      {Function()? onTap, Color? overrideColor}) {
+    final theme = Theme.of(context);
+    return Column(
+      children: [
+        _arrowLine(
+          context,
+          arrow: _postcardJourneyArrow(context),
+          child: Text(
+            distanceFormatter.format(
+                distance: travelInfo.getDistance(), prefix: "+"),
+            style: theme.textTheme.moMASans400Black12.copyWith(
+              color: overrideColor ?? MoMAColors.moMA12,
+              fontSize: 10,
+            ),
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.symmetric(vertical: 10),
+          child: _locationAddress(context,
+              address: travelInfo.to.address ?? "",
+              index: travelInfo.index + 1,
+              onTap: onTap),
+        ),
+      ],
+    );
+  }
+
+  Widget _webTravelWidget(TravelInfo travelInfo,
+      {Function()? onTap, Widget? arrow}) {
+    const overrideColor = AppColor.auQuickSilver;
+    final defaultArrow = _postcardWebUserArrow(context);
+    return _arrowLine(context,
+        arrow: arrow ?? defaultArrow,
+        child: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 10),
+          child: _locationAddress(
+            context,
+            index: travelInfo.index + 1,
+            address: travelInfo.to.address ?? "",
+            overrideColor: overrideColor,
+          ),
+        ),
+        onTap: onTap);
+  }
+
+  Widget _postcardJourneyArrow(BuildContext context) {
+    return SizedBox(
+      width: 9.5,
+      child: Align(
+        alignment: Alignment.topRight,
+        child: SvgPicture.asset(
+          "assets/images/arrow-50-head.svg",
+          fit: BoxFit.fitHeight,
+          height: 50,
+          width: 9,
+        ),
+      ),
+    );
+  }
+
+  Widget _postcardWebUserArrowWithHead(BuildContext context) {
+    return SizedBox(
+      width: 9.5,
+      child: Align(
+        alignment: Alignment.topRight,
+        child: SvgPicture.asset(
+          "assets/images/arrow-80-head.svg",
+          fit: BoxFit.fitHeight,
+          height: 80,
+          width: 9,
+        ),
+      ),
+    );
+  }
+
+  Widget _postcardWebUserArrow(BuildContext context) {
+    return SizedBox(
+      width: 9,
+      child: Align(
+        alignment: Alignment.topCenter,
+        child: SvgPicture.asset(
+          "assets/images/arrow-80.svg",
+          fit: BoxFit.fitHeight,
+          height: 80,
+        ),
+      ),
+    );
+  }
+
+  Widget _sendingTripItem(BuildContext context, AssetToken asset, int index) {
+    final theme = Theme.of(context);
+    return Column(
+      children: [
+        Row(
```

Comment:
can use `crossAxisAlignment: CrossAxisAlignment.start` for the Column & remove the Row wrapping outside?